packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -539,4 +620,15 @@ class HeroController extends NavigatorObserver {
   void _handleFlightEnded(_HeroFlight flight) {
     _flights.remove(flight.manifest.tag);
   }
+
+  static final HeroFlightShuttleBuilder _defaultHeroFlightShuttleBuilder = (
+    BuildContext flightContext,
+    Animation<double> animation,
+    HeroFlightDirection flightDirection,
+    BuildContext fromHeroContext,
+    BuildContext toHeroContext,
+  ) {
+    final Hero toHero = toHeroContext.widget;
+    return toHero.child;
+  };
 }
```

Comment:
regardless of whether you rename the new fields or create a diagram, please make sure to add significant documentation to this class to explain everything that's going on. Given the changes here, the class docs shouldn't have remained the same. :-)

Reply:
Changed the class dartdoc